safe_transaction_service/history/indexers/erc20_events_indexer.py

```diff
@@ -173,3 +182,40 @@ def process_elements(
             return range(
                 result_erc20 + result_erc721
             )  # TODO Hack to prevent returning `TokenTransfer` and using too much RAM
+
+    def get_almost_updated_addresses(
+        self, current_block_number: int
```

Comment:
`current_block_number` is not used in this function.

Reply:
Yes, but's inheriting from the parent class and must define it 😉